### PR TITLE
setting: update text for new 3rd party options

### PR DESCRIPTION
**Pull request by shawaj**
Update the text to refer to all 3rd party devices.

Change-type: patch
Signed-off-by: Aaron Shaw <shawaj@gmail.com>

### CODE REVIEW

**Review by matthewcarlreetz — COMMENTED** [review]
Thanks. Good catch on this one. 

@cokes518 We'll need to update the translations as well.

File: src/locales/en.ts [hunk 1]
```diff
@@ -591,7 +591,7 @@ export default {
     pairing: {
       title: 'Hotspot Pairing Required',
       subtitle:
-        'Press the button on your Helium Hotspot, or power-cycle your RAK Hotspot Miner.',
+        'Press the button or power-cycle your Helium or 3rd party Hotspot.',
```
Comment (cokes518) [inline src/locales/en.ts:594]: this implies that you need to power cycle the helium hotspot.

Recommended change ->
"Place your Hotspot in Bluetooth pairing mode."

Comment (owenselles) [reply src/locales/en.ts:594]: agree with this suggested change

Comment (tyler-whitman) [reply src/locales/en.ts:594]: @shawaj do you want to update this? Or want us to do it?

**Comment by tyler-whitman** [comment]
Closing as I made the change in https://github.com/helium/hotspot-app/pull/368

**Pull request closed without merging**